nixos/modules/services/networking/birdsitelive.nix

```diff
@@ -0,0 +1,442 @@
+                FollowersWhiteListing = mkOption {
+                  type = with types; listOf str;
+                  default = [ ];
+                  description = mdDoc "List of Fediverse user or instance patterns to allow as followers.";
+                  apply = concatOrNull;
+                };
+
+                FollowersBlackListing = mkOption {
+                  type = with types; listOf str;
+                  default = [ ];
+                  description = mdDoc "List of Fediverse user or instance patterns to disallow as followers.";
+                  apply = concatOrNull;
+                };
+
+                TwitterAccountsWhiteListing = mkOption {
+                  type = with types; listOf str;
+                  default = [ ];
+                  description = mdDoc "List of Twitter handles to allow following.";
+                  apply = concatOrNull;
+                };
+
+                TwitterAccountsBlackListing = mkOption {
+                  type = with types; listOf str;
+                  default = [ ];
+                  description = mdDoc "List of Twitter handles to disallow following.";
+                  apply = concatOrNull;
+                };
+              };
+            };
+          };
+        };
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    warnings = let mod = cfg.settings.Moderation; in
+      lib.optional (mod.FollowersWhiteListing != null && mod.FollowersBlackListing != null ||
+        mod.TwitterAccountsWhiteListing != null && mod.TwitterAccountsBlackListing != null) ''
+        If both both a whitelist and a blacklist are set for Fediverse or Twitter
+        accounts, only the whitelist will be used.
+      '';
+
+    users = {
+      users."${cfg.user}" = {
+        description = "BirdsiteLIVE user";
+        group = cfg.group;
+        isSystemUser = true;
+      };
+      groups."${cfg.group}" = { };
+    };
+
+    systemd.services.birdsitelive-config = {
+      description = "Generate BirdsiteLIVE Twitter bridge configuration";
+      reloadTriggers = [ configFile ] ++ secretPaths;
+
+      serviceConfig = {
+        Type = "oneshot";
+        RemainAfterExit = true;
+
+        RuntimeDirectory = "birdsitelive";
+        RuntimeDirectoryMode = "0750";
+
+        ExecStart = [
+          "${reloadScript}/bin/birdsiteliveReload"
+          "-${managerReloadScript}/bin/bslManagerReload"
+        ];
+        ExecReload = [
+          "${reloadScript}/bin/birdsiteliveReload"
+          "-${managerReloadScript}/bin/bslManagerReload"
+        ];
+      };
+    };
+
+    systemd.services.birdsitelive = {
+      confinement = {
+        enable = true;
+        packages = with pkgs; [
+          dotnetCorePackages.aspnetcore_3_1
+        ];
+      };
+
+      description = "BirdsiteLIVE Twitter to ActivityPub bridge";
+      bindsTo = [ "birdsitelive-config.service" ];
+      after = [ "birdsitelive-config.service" "network-online.target" "postgresql.service" ];
```

Comment:
```suggestion
      after = [ "birdsitelive-config.service" "network.target" "network-online.target" "postgresql.service" ];
```

Reply:
Is it necessary to explicitly list `network.target` if `network-online.target` already ordered after it?

Reply:
network-online.target does not necessarily mean there is an internet connection, only the link is up and ready. Also it is good practice to list both because network-online.target might be disabled because it is prone to delaying boot.

Reply:
Thank you for the explanation.